src/rpc/rawtransaction.cpp

```diff
@@ -66,7 +66,9 @@ UniValue TxJoinSplitToJSON(const CTransaction& tx) {
         UniValue joinsplit(UniValue::VOBJ);
 
         joinsplit.push_back(Pair("vpub_old", ValueFromAmount(jsdescription.vpub_old)));
+        joinsplit.push_back(Pair("vpub_oldZat", jsdescription.vpub_old));
```

Comment:
Fix problem reported here https://github.com/zcash/zcash/pull/3972#issuecomment-513510051 (but wanted to get this fix in asap, will rebase later PRs on this PR).